pages.id/common/adb-shell.md

```diff
@@ -0,0 +1,36 @@
+# adb shell
+
+> Android Debug Bridge Shell: Menjalankan perintah shell jarak jauh pada emulator Android atau perangkat Android terhubung.
+> Informasi lebih lanjut: <https://developer.android.com/studio/command-line/adb>.
+
+- Memulai shell interaktif jarak jauh di emulator/perangkat:
+
+`adb shell`
+
+- Mendapatkan semua properti dari emulator/perangkat:
+
+`adb shell getprop`
+
+- Mengembalikan semua izin runtime ke default:
+
+`adb shell pm reset-permissions`
+
+- Mencabut izin berbahaya dari sebuah aplikasi:
+
+`adb shell pm revoke {{package}} {{permission}}`
```

Comment:
Please translate the arguments for get this easier for non-English speakers. Like `package`, `permission` and `path/to/directory`, etc. in all pages.

Reply:
I have translated the arguments.